suite/tests/client-interface/drx-scattergather-aarch64.cpp

```diff
@@ -540,16 +636,54 @@ class input_data_t {
         // The base address is set to the 8th element in the region.
         return (1024 * offset) + (static_cast<size_t>(element_size) * 8);
     }
+};
 
-    template <typename T>
-    void
-    write_input_data(size_t offset, const std::array<T, 40> &input_data)
+class output_data_t : public test_memory_t {
+public:
+    output_data_t()
+        : test_memory_t()
     {
-        memcpy(region_start_addr(offset), input_data.data(),
-               input_data.size() * sizeof(T));
+        /*
+         * We set up nine pages of memory to use as input data for load instruction
+         * tests. The first page contains input data of different sizes and the rest
+         * of the pages are set up to fault when they are accessed. The tests use
+         * one of the first 4 regions as their base pointer, so when we want to
+         * force an instruction to fault (to test faulting behaviour) we can set the
+         * offset to 4096 which will always land in one of the faulting pages.
+         * +=====================================================+
+         * | Page   | Byte off | Region off |                    |
+         * +=====================================================+
+         * | page 0 |        0 |          0 | -ve offset data    |
```

Comment:
What is "ve"?

Reply:
"+ve" and "-ve" are abbreviations for "positive" and "negative" respectively
https://en.wiktionary.org/wiki/%2Bve